common/darwin/Classes/FlutterWebRTCPlugin.m

```diff
@@ -542,6 +542,11 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult) result
             }
             [dataChannels removeAllObjects];
         }
+
+        #if TARGET_OS_IPHONE
+                [[NSNotificationCenter defaultCenter] removeObserver:self];
```

Comment:
Here is for sure some formatting issue. Please use the same formatting as in edited file.